Gems/Prefab/PrefabBuilder/PrefabGroup/tests/PrefabBehaviorTests.cpp

```diff
@@ -379,7 +387,9 @@ namespace UnitTest
         AzToolsFramework::Prefab::Instance instance;
         ASSERT_TRUE(AzToolsFramework::Prefab::PrefabDomUtils::LoadInstanceFromPrefabDom(instance, *prefabDomRef));
         EXPECT_TRUE(IsChildOfParent(instance, "3", "2")); // Mesh entity is child of a transform entity
-        EXPECT_TRUE(IsChildOfParent(instance, "5", "2")); // Transform entity is child of another transform entity
+        EXPECT_TRUE(IsChildOfParent(instance, "6", "2")); // Transform entity is child of another transform entity
+        EXPECT_FALSE(IsChildOfParent(instance, "5", "4")); // First transform entity is not a child of the mesh entity
```

Comment:
Should this have been IsChildOfParent(instance, "5", "3")?

Reply:
Yes, thanks! Changed and test still passes :)